feeds/src/components/vis/info/Info.tsx

```diff
@@ -1,85 +1,85 @@
 import React from 'react'
 import { Icon } from 'antd'
-import CountDown from './CountDown.component'
-import Percent from './Percent.component'
-import Legend from './Legend.component'
-import TooltipQuestion from '../shared/TooltipQuestion'
+import Heartbeat from './Heartbeat'
+import Percent from './Percent'
+import Legend from './Legend'
+import TooltipQuestion from '../../shared/TooltipQuestion'
 import { humanizeUnixTimestamp, networkName, Networks } from 'utils'
 import { FeedConfig } from 'feeds'
 
 interface OwnProps {
-  options: FeedConfig
+  config: FeedConfig
 }
 
 interface StateProps {
-  currentAnswer: any
-  requestTime: any
-  minimumResponses: any
-  oracleResponse: any
-  oracles: any
-  updateHeight: any
-  pendingAnswerId: any
+  latestAnswer: string
+  latestRequestTimestamp: number
+  minimumAnswers: number
+  oracleAnswers: any
+  oracleList: Array<string>
+  latestAnswerTimestamp: any
+  pendingRoundId: any
 }
 
 export interface Props extends OwnProps, StateProps {}
 
-const NetworkGraphInfo: React.FC<Props> = ({
-  currentAnswer,
-  requestTime,
-  minimumResponses,
-  oracleResponse,
-  oracles,
-  updateHeight,
-  options,
-  pendingAnswerId,
+const Info: React.FC<Props> = ({
+  latestAnswer,
+  latestAnswerTimestamp,
+  latestRequestTimestamp,
+  minimumAnswers,
+  oracleAnswers,
+  oracleList,
+  config,
+  pendingRoundId,
 }) => {
-  const updateTime = updateHeight
-    ? humanizeUnixTimestamp(updateHeight, 'h:mm A')
+  const updateTime = latestAnswerTimestamp
+    ? humanizeUnixTimestamp(latestAnswerTimestamp, 'h:mm A')
     : '...'
 
-  const updateDate = updateHeight
-    ? humanizeUnixTimestamp(updateHeight, 'MMM Do YYYY')
+  const updateDate = latestAnswerTimestamp
+    ? humanizeUnixTimestamp(latestAnswerTimestamp, 'MMM Do YYYY')
     : '...'
 
   const getCurrentResponses = () => {
-    if (!oracleResponse) {
+    if (!oracleAnswers) {
       return '...'
     }
 
-    const responended = oracleResponse.filter((r: any) => {
-      return r.answerId >= pendingAnswerId
+    const responended = oracleAnswers.filter((r: any) => {
+      return r.answerId >= pendingRoundId
     })
 
-    return `${responended.length} / ${oracles && oracles.length}`
+    return `${responended.length} / ${oracleList && oracleList.length}`
```

Comment:
If `oracleList` is falsey here, wont it return a string that doesnt make sense?
Maybe it should be something like: 
```suggestion
    return oracleList && oracleList.length ? `${responended.length} / ${oracleList.length}` : 'N/A'
```

Reply:
Sounds good. I will use `ellipsis` as its loading indicator. I cannot think about scenario where we don't have any oracles.